### ISSUE TITLE
[Chore]: Reduce cost of mel-c, if possible

### ISSUE THREAD

**Issue by kieron-dev** [cloudfoundry/korifi#2096]
### Background

We have a bosh-lite director deployed on GCP. See cf-k8s-secrets/bosh-lite-cf/directors/mel-c.

It's on all the time, and currently running on a n1-standard-8 machine.

### Action to take

We should switch the machine type to e2 to reduce the running cost.

Other possibilities:
- look at the GCP monitoring to determine if we can reduce the CPU count and memory size. e2 allows custom sizing.
- introduce a concourse pipeline that destroys the director at night, and brings it back up in the morning
- see if we can automatically stop the machine at night and start it on weekday mornings
  - will bosh survive the restart?

### Impact

Save some money. But try not to spend more money doing it than the potential savings.

**Comment by kieron-dev** [comment]
We've switched the machine type to e2-custom with 8CPU / 16Gb RAM.

We tried stopping the machine, but can't get the bosh director to come back up again. If you run `lite-me-up deploy-director`, it will recover the director, but the deployments have no VMs and need fixing. So the whole process takes about as long as the initial clean deploy.

So we think scheduling mel-c creation in concourse in the morning, and deletion in the evening is a reasonable approach.
